src/components/sortingFunctions.js

```diff
@@ -10,13 +10,13 @@ export const getShoppingItemBackgroundStyles = (
   );
 
   if (daysBetweenCurrentAndPreviousPurchase >= daysLeftForNextPurchase * 2) {
-    return 'grey';
+    return '#D1D5DB';
   } else if (daysLeftForNextPurchase <= 7) {
-    return 'green';
+    return '#047857';
   } else if (daysLeftForNextPurchase < 30) {
-    return 'orange';
+    return '#10B981';
   } else {
-    return 'red';
+    return '#6EE7B7';
```

Comment:
Hmm this is a good idea. I was just wondering if it would not be more distinct to have totally different colors for all of the different cases? 

Or you could somehow indicate the meaning of the colors to the user?

Reply:
I am still not sure how we are going with this. Another suggestion I made in the other PR was to style the radio buttons in that color the items will appear in the list. But this will be a task for the following week (or rather I am already working on different upcoming styling parts) because this is only possible with the tailwind config which I did not have until now 😉